roles/common/defaults/main.yml

```diff
@@ -34,6 +34,7 @@ apt_packages_default:
   python3-mysqldb: "{{ apt_package_state }}"
   python3-pycurl: "{{ apt_package_state }}"
   unzip: "{{ apt_package_state }}"
+  cron: "{{ apt_package_state }}"
```

Comment:
Can you put this above `curl` in alphabetical order please? 👼

Reply:
Sure!